include/fuse_lowlevel.h

```diff
@@ -1968,26 +1970,29 @@ int fuse_session_mount(struct fuse_session *se, const char *mountpoint);
 int fuse_session_loop(struct fuse_session *se);
 
 #if FUSE_USE_VERSION < 32
-int fuse_session_loop_mt_31(struct fuse_session *se, int clone_fd);
-#define fuse_session_loop_mt(se, clone_fd) fuse_session_loop_mt_31(se, clone_fd)
+	int fuse_session_loop_mt_31(struct fuse_session *se, int clone_fd);
+	#define fuse_session_loop_mt(se, clone_fd) fuse_session_loop_mt_31(se, clone_fd)
+#elif FUSE_USE_VERSION < FUSE_MAKE_VERSION(3, 11)
+	int fuse_session_loop_mt_32(struct fuse_session *se, struct fuse_loop_config *config);
+	#define fuse_session_loop_mt(se, config) fuse_session_loop_mt_32(se, config)
 #else
-#if (!defined(__UCLIBC__) && !defined(__APPLE__))
-/**
- * Enter a multi-threaded event loop.
- *
- * For a description of the return value and the conditions when the
- * event loop exits, refer to the documentation of
- * fuse_session_loop().
- *
- * @param se the session
- * @param config session loop configuration 
- * @return see fuse_session_loop()
- */
-int fuse_session_loop_mt(struct fuse_session *se, struct fuse_loop_config *config);
-#else
-int fuse_session_loop_mt_32(struct fuse_session *se, struct fuse_loop_config *config);
-#define fuse_session_loop_mt(se, config) fuse_session_loop_mt_32(se, config)
-#endif
+	#if (!defined(__UCLIBC__) && !defined(__APPLE__))
```

Comment:
There seem to be  a lot of spurious changes here (maybe space vs tab). Could you get rid of those?

Reply:
Hmm, it is a bit hard to read the conditions without any indentation - I had added that on purpose.

```
#if FUSE_USE_VERSION < 32
	int fuse_session_loop_mt_31(struct fuse_session *se, int clone_fd);
	#define fuse_session_loop_mt(se, clone_fd) fuse_session_loop_mt_31(se, clone_fd)
#elif FUSE_USE_VERSION < FUSE_MAKE_VERSION(3, 11)
	int fuse_session_loop_mt_32(struct fuse_session *se, struct fuse_loop_config *config);
	#define fuse_session_loop_mt(se, config) fuse_session_loop_mt_32(se, config)
#else
	#if (!defined(__UCLIBC__) && !defined(__APPLE__))
	/**
	 * Enter a multi-threaded event loop.
	 *
	 * For a description of the return value and the conditions when the
	 * event loop exits, refer to the documentation of
	 * fuse_session_loop().
	 *
	 * @param se the session
	 * @param config session loop configuration
	 * @return see fuse_session_loop()
	 */
		int fuse_session_loop_mt(struct fuse_session *se, struct fuse_loop_config *config);
	#else
		int fuse_session_loop_mt_311(struct fuse_session *se, struct fuse_loop_config *config);
		#define fuse_session_loop_mt(se, config) fuse_session_loop_mt_311(se, config)
	#endif
#endif
```
vs

```
#if FUSE_USE_VERSION < 32
int fuse_session_loop_mt_31(struct fuse_session *se, int clone_fd);
#define fuse_session_loop_mt(se, clone_fd) fuse_session_loop_mt_31(se, clone_fd)
#elif FUSE_USE_VERSION < FUSE_MAKE_VERSION(3, 11)
int fuse_session_loop_mt_32(struct fuse_session *se, struct fuse_loop_config *config);
#define fuse_session_loop_mt(se, config) fuse_session_loop_mt_32(se, config)
#else
#if (!defined(__UCLIBC__) && !defined(__APPLE__))
/**
 * Enter a multi-threaded event loop.
 *
 * For a description of the return value and the conditions when the
 * event loop exits, refer to the documentation of
 * fuse_session_loop().
 *
 * @param se the session
 * @param config session loop configuration
 * @return see fuse_session_loop()
 */
int fuse_session_loop_mt(struct fuse_session *se, struct fuse_loop_config *config);
#else
int fuse_session_loop_mt_311(struct fuse_session *se, struct fuse_loop_config *config);
#define fuse_session_loop_mt(se, config) fuse_session_loop_mt_311(se, config)
#endif
#endif
```